src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  let PRICE_ALL = PRICE_PER_DAY * days;
```

Comment:
```suggestion
  const currentPrice = PRICE_PER_DAY * days;
```
Prefer const over let
And variable names should be in camelCase